vertical-pod-autoscaler/enhancements/8459-memory-per-cpu/README.md

```diff
@@ -0,0 +1,167 @@
+# AEP-8459: MemoryPerCPU
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+  - [API Changes](#api-changes)
+  - [Behavior](#behavior)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+    - [How can this feature be enabled / disabled in a live cluster?](#how-can-this-feature-be-enabled--disabled-in-a-live-cluster)
+  - [Kubernetes Version Compatibility](#kubernetes-version-compatibility)
+  - [Validation](#validation)
+  - [Test Plan](#test-plan)
+- [Implementation History](#implementation-history)
+- [Future Work](#future-work)
+- [Alternatives](#alternatives)
+  <!-- /toc -->
+
+## Summary
+
+This AEP proposes a new feature to allow enforcing a fixed memory-per-CPU ratio (`memoryPerCPU`) in Vertical Pod Autoscaler (VPA) recommendations.  
+The feature is controlled by a new alpha feature gate `MemoryPerCPURatio` (default off).
+
+## Motivation
+
+Many workloads scale their memory requirements proportionally to CPU, but today VPA generates independent CPU and memory recommendations. This can lead to skewed configurations — for example, too much memory for a small CPU allocation, or too little memory for a high CPU allocation.
```

Comment:
Can this be expanded on?

I'm not sure I understand the reasoning for this feature. 
I haven't yet come across a workload that scales memory proportionally to CPU. And even if this was the case, won't the recommendation of the VPA eventually be the correct ratio?

Reply:
That's good to know!
Is it possible to get more feedback from more users so we have an idea of how useful this is?